scripts/Interface/ui/drill/model/configurations.py

```diff
@@ -221,7 +221,8 @@ class RundexSettings(object):
                 "DeltaQ",
                 "IQxQyLogBinning",
                 "OutputPanels",
-                "WavelengthRange"
+                "WavelengthRange",
+                "StitchReferenceIndex"
```

Comment:
Thanks for the changes, one more small request, although unrelated.
Could you also add the ClearCorrected2DWorkspace here?

Reply:
Yes, done.